lms/templates/page-builder/elements/_courses-listing.html

```diff
@@ -7,7 +7,7 @@
 <section class="amc--element--courses-listing courses-listing ${options['tile-type']}--parent-element amc--style-classes ${options['text-alignment']}">
   % if not courses is UNDEFINED:
     % for course in courses:
```

Comment:
```suggestion
    % for course in courses[:int(options['num-of-courses'])]:
```

[As seen in Stackoverflow](https://stackoverflow.com/a/10897354).

Reply:
@grozdanowski do we need a default value for `options['num-of-courses']`? or that's something that's always provided?

Reply:
It's always provided.